jstests/libs/backup_utils.js

```diff
@@ -113,13 +113,13 @@ function _copyFiles(files, dbpath, destinationDirectory, copyFileHelper) {
 
 function _copyFileHelper(absoluteFilePath, sourceDbPath, destinationDirectory) {
     // Ensure the dbpath ends with an OS appropriate slash.
+    let separator = '/';
+    if (_isWindows()) {
+        separator = '\\';
+    }
```

Comment:
Can the ternary operator be used here to reduce the whole block with a single line?